### PR TITLE
Request other community engagement

### PR DESCRIPTION
**Pull request by azabeli**
Issue:  Create a basic form that a student will fill in to request an experience completed outside of the college. The page should include: the student's name, the name of the position they worked/volunteered for, the total hours they worked/volunteered, the total weeks they worked/volunteered, a description of the work, the term the experience is going to be credited in,  and a place to upload a file from a supervisor that verifies the student did work/volunteer the amount they said. It should also have 3 statuses associated with an "Other" engagement.  

Fix: We created and rendered the page with the name of the student and a form to fill in other information like the name of the position they worked/volunteered for, total hours they worked/volunteered, the total weeks they worked/volunteered, description of the work,  and we also created a place to upload a file from a supervisor that verifies the student did work/volunteer the number of hours they mentioned.

Test: load the following route to see the page: /cceMinor/username/requestOtherCommunityEngagement
When you load the page, you will see the form on the page where you can fill in the relevant information about the request and other community engagement. and upload the doc file from your supervisor. 

resolving issue #990

### CODE REVIEW

File: app/controllers/minor/routes.py [hunk 1]
```diff
@@ -30,12 +34,22 @@ def removeCommunityEngagement(username):
...
-def requestOtherCommunityEngagement(username):
+@minor_bp.route('/cceMinor/<username>/requestOtherCommunityEngagement', methods=['GET'])
+def requestOtherEngagement(username):
     """
         Load the "request other" form and submit it.
     """
-    pass
+    user = User.get_by_id(username)
+    terms = selectSurroundingTerms(g.current_term)
+    # filepaths = handleFileSelection()
```
Comment (gahimbaref) [inline app/controllers/minor/routes.py]: remove unused code

**Review by gahimbaref — COMMENTED** [review]
image

- "Other Credit" could be made a bit more descriptive 
-  a user can input anything in the hours field, I think it should be limited to numbers (not letters)
- company/organization box seems a little out of place 
- Not sure the difference between Activity vs Activity/Service, they are 2 different fields)

**Comment by gahimbaref** [comment]
image

It says page not found after submitting the form `Send Other Request`

**Comment by andersoncedu** [comment]
image

There might be a problem here.

File: app/models/otherExperience.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from app.models import *
+from app.models.term import Term
+
+class OtherExperience(baseModel):
```
Comment (BrianRamsay) [inline app/models/otherExperience.py:4]: Add 'CCE' to the name somewhere

File: app/models/otherExperience.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from app.models import *
+from app.models.term import Term
+
+class OtherExperience(baseModel):
+    activity = CharField()
+    term = ForeignKeyField(Term)
+    hours = IntegerField()
+    weeks = IntegerField()
+    service = CharField()
```
Comment (BrianRamsay) [inline app/models/otherExperience.py:9]: 'description' is probably a better name for this, and matches what you have in the UI

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

**Pull request closed without merging**